mlir/include/mlir/Interfaces/LinkageInterfaces.td

```diff
@@ -43,6 +43,14 @@ def GlobalValueLinkageOpInterface : OpInterface<"GlobalValueLinkageOpInterface">
       /*methodName=*/  "getLinkage",
       /*args=*/        (ins)
     >,
+    InterfaceMethod<
+      /*desc=*/        "Sets the linkage of the operation",
+      /*returnType=*/  "void",
+      /*methodName=*/  "setLinkage",
+      /*args=*/(ins "::mlir::link::Linkage":$linkage),
+      /*methodBody=*/"",
+      /*defaultImplementation=*/[{ }]
```

Comment:
I would like this not to stay empty. As it might be source of bugs if you forget to implement it.

Reply:
Good catch! That's an oversight of me. I'll fix it.

Reply:
I'm struggling a bit to get this working. To clarify, did you intend for me to implement either of these scenarios?
1. `/*defaultImplementation=*/""` 
2. `/*defaultImplementation=*/[{ actual default implementation }]`
Right now, no matter how I do this (seemingly) I get redeclaration errors. I think this might have something to do:

```
def LLVM_GlobalOp : LLVM_Op<"mlir.global",
    [IsolatedFromAbove, SingleBlockImplicitTerminator<"ReturnOp">, Symbol,
    GlobalVariableLinkageOpInterface,
    DeclareOpInterfaceMethods<GlobalValueLinkageOpInterface>,
    DeclareOpInterfaceMethods<GlobalVariableLinkageOpInterface>]> {
```

Any hints?

Reply:
I wanted you to drop 

```
/*methodBody=*/"",
      /*defaultImplementation=*/[{ }]
```

and only add definition on missing classes